package.json

```diff
@@ -42,8 +42,8 @@
     "alpheios-client-adapters": "github:alpheios-project/client-adapters#v0.0.9",
     "alpheios-data-models": "github:alpheios-project/data-models#v2.5.1",
     "alpheios-experience": "github:alpheios-project/experience#v1.1.0",
-    "alpheios-inflection-tables": "github:alpheios-project/inflection-tables#v1.5.2",
-    "alpheios-node-build": "github:alpheios-project/node-build#v0.6.2",
+    "alpheios-inflection-tables": "github:alpheios-project/inflection-tables#v1.5.3",
```

Comment:
there is no functional difference between these two releases. Some confusion I had looking at the release history for a bug last week caused me to do another build and release, so this should have no effect.